lib/android.js

```diff
@@ -15,10 +15,10 @@ checks.push(new EnvVarAndPathCheck('JAVA_HOME'));
 class JavaOnPathCheck extends DoctorCheck {
   async diagnose () {
     let javaHomeBin = path.resolve(process.env.JAVA_HOME, 'bin');
-    if (process.env.PATH.indexOf(javaHomeBin)) {
-      return nok('Bin directory of $JAVA_HOME is not set');
+    if (process.env.PATH.indexOf(javaHomeBin) + 1) {
```

Comment:
I've been thinking about this check. I think we also need to take into account the possibility of having `$JAVA_HOME/bin` in the `PATH`, instead of the full path.

```js
if (process.env.PATH.indexOf(javaHomeBin) === -1 && process.env.PATH.indexOf(`$JAVA_HOME${path.sep}bin`) === -1) {
  return nok('Bin directory for $JAVA_HOME is not set');
}
```